eth/vm/base.py

```diff
@@ -577,19 +578,16 @@ def validate_header(self,
                     f"- parent : {parent_header.timestamp}. "
                 )
 
-            if check_seal:
-                try:
-                    self.validate_seal(header)
-                except ValidationError as exc:
-                    self.cls_logger.warning(
-                        "Failed to validate seal on header: %r. Error: %s",
-                        header.as_dict(),
-                        exc,
-                    )
-                    raise
-
     def validate_seal(self, header: BlockHeaderAPI) -> None:
-        self._consensus.validate_seal(header)
+        try:
+            self._consensus.validate_seal(header)
+        except ValidationError as exc:
+            self.cls_logger.warning(
```

Comment:
I moved this here because I think the standard logging on a failed seal is a net good for debugging etc.

Reply:
So one potentially negative effect of this is that if someone sent us an invalid header in trinity and it hits this method then trinity will output this warning message to the console.  Is that what we want?

Reply:
> Is that what we want?

Hm, not sure. Geth does print out warnings about bad *blocks*, and it causes some concern/confusion.

Reply:
What if this default log entry would only go into `debug` or even `debug2` instead?

Reply:
`debug` seems appropriate.